### PR TITLE
Edit advertisement backend

### PR DESCRIPTION
**Pull request by kalyanpullela**
Edit the advertisement backend to have editAdvertisement and deleteAdvertisement

### CODE REVIEW

File: api/models/Advertisement.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    createDate: {
+      type: Date,
+      required: true,
+      default: Date.now
+    },
+    expireDate: {
+      type: Date
+    }
+  },
+  { collection: 'Advertisements' }
```
Comment (ThaiQ) [inline api/models/Advertisement.js:19]: lets stick with singular @kalyanpullela

File: src/APIFunctions/Advertisement.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+ */
+export async function createNewAdvertisement(newAdvertisement, token) {
+  let status = new ApiResponse();
+  const advertisementToAdd = {
+    eventDate: newAdvertisement.advertisementStartDate,
+    eventDate: newAdvertisement.advertisementEndDate,
+    imageURL: newEvent.imageURL
+  };
+  await axios
+    .post('api/Advertisement/createAdvertisements', { token, ...advertisementToAdd })
```
Comment (ThaiQ) [inline src/APIFunctions/Advertisement.js:41]: singular

File: src/APIFunctions/Advertisement.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+ * the event
+ * @param {(string|undefined)} eventToUpdate.eventCategory - The updated
+ * eventCategory of the event
+ * @param {(string|undefined)} eventToUpdate.imageURL - An updated image URL of
+ * the event
+ * @param {string} token - The user's jwt token for authentication
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+ /*
```
Comment (ThaiQ) [inline src/APIFunctions/Advertisement.js:75]: what happened here? We should delete these comments before merging to dev

**Comment by ThaiQ** [comment]
rebase to the latest dev-commit my dude

**Comment by evanugarte** [comment]
this pr is going stale, closing it until we can pick up work on it again.

**Pull request closed without merging**